django/thunderstore/social/tests/test_complete_login.py

```diff
@@ -216,7 +216,7 @@ def test_valid_request(api_client: APIClient, provider: str) -> None:
     response = _post(api_client, url)
 
     assert (response.status_code) == 200
-    assert (response.json()["session_id"]) == "TODO"
+    assert isinstance(response.json()["session_id"], str)
```

Comment:
This test could validate the session ID to be valid on top of checking it to be an str

Reply:
Not sure what you meant by "valid" but it now checks the session ID exists in the db.

Reply:
Yep that's what I meant, nice 👍